Dockerfile.Konflux

```
@@ -1,7 +1,8 @@
 # Copyright Contributors to the Open Cluster Management project
 # Licensed under the Apache License 2.0
 
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.25.7 AS builder
+FROM brew.registry.redhat.io/
```

Hadolint (2.14.0)

[error] expecting the image name with a tag

(DL1000)